feathr_project/feathr/_feature_registry.py

```diff
@@ -431,17 +431,18 @@ def _read_config_from_workspace(self, workspace_path: str):
             os.path.join(workspace_path, "feature_conf", '*.conf'))
         logger.info("Reading feature configuration from {}",
                     feature_config_paths)
-        feature_config_path = feature_config_paths[0]
-        self.feathr_feature_config = ConfigFactory.parse_file(
-            feature_config_path)
-        # print(self.feathr_feature)
-        with open(feature_config_path, "r") as f:
-            raw_hocon_feature_definition_config = f.read()
-
-        # get feature join config file
-        feature_join_paths = glob.glob(os.path.join(
-            workspace_path, "feature_join_conf", '*.conf'))
-        logger.info("Reading feature join configuration from {}",
+        if len(feature_config_paths) > 0:
```

Comment:
if there are no configs, log an warning?

Reply:
This is a valid case, so I don't think a warning is necessary.